main.tf

```diff
@@ -45,6 +45,10 @@ resource "aws_api_gateway_stage" "stage" {
     format          = jsonencode(var.log_format)
   }
 
+  tags = {
+    global_cwl_log_arn = var.enable_global_apigw_logging ? aws_api_gateway_account.api_gateway_account[0].cloudwatch_role_arn : ""
+  }
```

Comment:
@jaezeu how this tag will be used ?

Reply:
@niroz89 this is to force implicit dependency so that the stage will be created after the global `aws_api_gateway_account` cloudwatch role arn is updated.

Reply:
I think `depends_on = [aws_api_gateway_account.api_gateway_account]` might be sufficient.